src/Storages/MergeTree/MergeTreeIndexAnnoy.cpp

```diff
@@ -154,36 +154,45 @@ void MergeTreeIndexAggregatorAnnoy<Distance>::update(const Block & block, size_t
 
     if (const auto & column_array = typeid_cast<const ColumnArray *>(column_cut.get()))
     {
-        const auto & data = column_array->getData();
-        const auto & array = typeid_cast<const ColumnFloat32 &>(data).getData();
+        const auto & column_array_data = column_array->getData();
+        const auto & column_arary_data_float_data = typeid_cast<const ColumnFloat32 &>(column_array_data).getData();
```

Comment:
So much data
```suggestion
        const auto & column_arary_data_float = typeid_cast<const ColumnFloat32 &>(column_array_data).getData();
```

Reply:
There is some logic behind the naming. Arrays and tuples are nested structures, so, yes, there will be a few `data` in variables.